apps/microsoft-teams/frontend/src/components/config/ChannelSelection/ChannelSelection.tsx

```diff
@@ -44,7 +42,7 @@ const ChannelSelection = (props: Props) => {
           <TextInput
             id="selected-channel"
             isDisabled={true}
-            value={getChannelName(notification.channelId, mockChannels, channelSelection.notFound)}
+            value={getChannelName(notification.channelId, channels, channelSelection.notFound)}
```

Comment:
@whitelisab we can get rid of this function after adding `channelName` to `Notification` and pass `notification.channelName` as the value?

Reply:
going to update this in a follow up PR 💪